README.md

```diff
@@ -26,58 +28,33 @@
 5. Install and update [Android Studio](https://developer.android.com/studio/index.html)
 * React Native Android installation [instructions](https://facebook.github.io/react-native/docs/getting-started.html)
 
-## Installation:
-
-1. Clone this repo
-
-2. In your terminal, change your directory to the root of the sample project you want to run.
-
-3. Run the command:
- 
-        npm install
-    to install required node modules
-
-4. For iOS:
-  * `cd ios`
-  * `pod install`
-
-4. For Android:
-  * `react-native link`
-  *  Add the following to your project's `build.gradle` file: 
+## Installation
 
-            maven {
-                url "http://tokbox.bintray.com/maven"
-            }
+To install this package in your project, run the following command:
 
-## Configuring the application
-
-Before running the application, you need to configure it to use the API key for your OpenTok project, along with an OpenTok session ID and token. For development purposes, you can obtain a session ID and token by navigating to your [TokBox account](https://tokbox.com/account/#/) page, selecting a project, and scrolling to the bottom of the page where it says `Generate Token`.
-
-Open the `src/App.js` file in your project and set the `this.apiKey`, `this.sessionId`, and `this.token` values to the API key, session ID, and token you obtained from your TokBox account:
 ```
-// Set Credentials
-this.apiKey = '';    // Add your API key.
-this.sessionId = ''; // Add the session ID.
-this.token = '';     // Add the token.
+npm install opentok-react-native --save
+react-native link opentok-react-native
 ```
-An OpenTok session connects different clients letting them share audio-video streams and send messages. Clients in the same session can include iOS, Android, and web browsers.
-
-For testing, you can use a session ID and token generated at your TokBox account page. However, the final application should obtain these values using the OpenTok server SDKs. For more information, see the OpenTok [server SDK guides](https://tokbox.com/developer/sdks/server/) on session and token creation.
-
-## Running the application
 
-#### For Android
+### For iOS:
 
-1. In the root directory of this project, run `npm run android`.
+* `cd ios`
+* `pod install`
 
-##### Note: If you're running the app on the simulator, you will see a simulation for your publisher because the simulator doesn't have a camera.
+2. For Android:
 
-#### For iOS
+* `react-native link opentok-react-native`
+*  Add the following to your project's `build.gradle` file:
```

Comment:
They should not run `react-native link opentok-react-native` after they've already done it for iOS. Maybe add a disclaimer mentioning that they don't have to do it if they already have for iOS.

Reply:
oops, my bad
Yeah, adding a disclaimer sounds like a good idea.